northstar_tests/src/runtime.rs

```diff
@@ -65,7 +65,16 @@ pub fn timeout_on<R>(f: impl Future<Output = R>) -> Result<R> {
 }
 
 /// A running instance of northstar.
-pub struct Runtime(runtime::Runtime);
+pub struct Runtime(Option<runtime::Runtime>);
+
+impl Drop for Runtime {
```

Comment:
drop takes a &mut self. Why is this needed to be an `Option`? Because you want to move it into the `block_on`?

Reply:
Yes, exactly this :arrow_up: